tests/pool_test.go

```diff
@@ -521,6 +521,19 @@ var _ = Describe("[sig-compute]VirtualMachinePool", decorators.SigCompute, func(
 			Expect(vm.OwnerReferences).To(BeEmpty())
 		}
 		Expect(err).ToNot(HaveOccurred())
+
+		// Remove the finalizer
+		for _, vm := range vms.Items {
+			vm, err := virtClient.VirtualMachine(vm.Namespace).Get(context.Background(), vm.Name, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			if vm.Finalizers == nil {
+				continue
+			}
+
+			vm.Finalizers = []string{}
+			_, err = virtClient.VirtualMachine(vm.Namespace).Patch(context.Background(), vm.Name, types.JSONPatchType, []byte("[{ \"op\": \"remove\", \"path\": \"/metadata/finalizers\" }]"), metav1.PatchOptions{})
+			Expect(err).ToNot(HaveOccurred())
+		}
```

Comment:
I think this is not a solution.
We should remove the finalizer when orphan deletion is executed.
Please, consider this:
https://github.com/kubevirt/kubevirt/commit/1d4f52c49d9f253849c58a6a33915d336ff8cfb6